src/vm/jitinterface.cpp

```diff
@@ -489,12 +489,14 @@ CEEInfo::ConvToJitSig(
         IfFailThrow(sig.GetCallingConvInfo(&data));
         sigRet->callConv = (CorInfoCallConv) data;
 
+#ifdef _TARGET_UNIX_
```

Comment:
`_TARGET_UNIX_` is JIT-specific define. Outside of JIT, we have `PLATFORM_UNIX`.

Reply:
Thank you for catching this.